src/H5Pdxpl.c

```diff
@@ -2456,6 +2467,39 @@ H5Pget_no_selection_io_cause(hid_t plist_id, uint32_t *no_selection_io_cause /*o
     FUNC_LEAVE_API(ret_value)
 } /* end H5Pget_no_selection_io_cause() */
 
+/*-------------------------------------------------------------------------
+ * Function:	H5Pget_actual_selection_io_mode
+ *
+ * Purpose:	    Retrieves actual selection I/O mode
+ *
+ * Return:	    Non-negative on success/Negative on failure
+ *
+ * Programmer:	Vailin Choi
+ *              April 27, 2023
+ *-------------------------------------------------------------------------
+ */
+herr_t
+H5Pget_actual_selection_io_mode(hid_t plist_id, uint32_t *actual_selection_io_mode /*out*/)
+{
+    H5P_genplist_t *plist;
+    herr_t          ret_value = SUCCEED; /* return value */
+
+    FUNC_ENTER_API(FAIL)
+    H5TRACE2("e", "ix", plist_id, actual_selection_io_mode);
```

Comment:
Should this be `i*Iu` for the trace macro? Looks like that's used for both unsigned and uint32_t

Reply:
This is what bin/trace spits out. It might be an error but it seems to be the same for other similar functions.

Reply:
It seems inconsistent based on https://github.com/HDFGroup/hdf5/blob/b916ce2419da45f7a51503e3bf2774e71f7db815/src/H5D.c#L1204

Reply:
It might be getting confused by the "/* out */"

Reply:
This looks like a bug in the bin/trace script that affects many existing API functions. I think the fix is outside the scope of this PR

Reply:
Filed issue #3733